_includes/merchant-authenticated-consumer.md

```diff
@@ -51,5 +51,18 @@ Content-Type: application/json
 |          | └──➔&nbsp;`socialSecurityNumber` | `string` | The consumers social security number. Must be part of what you get from your authentication process. |
 |          | └──➔&nbsp;`countryCode`          | `string` | The countrycode of the consumer.                                                                     |
 
+When the consumer is authenticated by the Merchant, some payment instruments
+will allow a more frictionless payment process. Invoice One-Click payments does
+not require the last 4/5 digits in the payers social security number (SSN).
+Below you can see an example of how the payment window looks like with the
+One-Click payment without SSN to the left. And the payer can press the pay
+button directly. To the right, the payment window
+from the response and request from above with the required SSN number.
```

Comment:
```suggestion
When the consumer is authenticated by the Merchant, some payment instruments
will allow a more frictionless payment process. Invoice One-Click payments does
not require the last 4/5 digits in the payer's social security number (SSN), for
instance.

Below you can see an example of how the payment window looks like with and 
without Merchant Authenticated Consumer. To the left, the SSN is removed
altogether because the payer is authenticated by the Merchant, and to the right
the last 4/5 digits of the SSN required as usual.
```